src/DataProtection/DataProtection/src/IDataProtectionBuilder.cs

```diff
@@ -15,14 +15,14 @@ namespace Microsoft.AspNetCore.DataProtection
     /// <remarks>
     /// <para>
     /// If the developer changes the at-rest key protection mechanism, it is intended that
-    /// he also change the key storage location, and vice versa. For instance, a call to
+    /// they also change the key storage location, and vice versa. For instance, a call to
     /// <see cref="DataProtectionBuilderExtensions.ProtectKeysWithCertificate(IDataProtectionBuilder,string)" /> should generally be accompanied by
     /// a call to <see cref="DataProtectionBuilderExtensions.PersistKeysToFileSystem(IDataProtectionBuilder,DirectoryInfo)"/>, or exceptions may
     /// occur at runtime due to the data protection system not knowing where to persist keys.
     /// </para>
     /// <para>
     /// Similarly, when a developer modifies the default protected payload cryptographic
-    /// algorithms, it is intended that he also select an explitiy key storage location.
+    /// algorithms, it is intended that the key storage location is explitily set.
```

Comment:
Actually, we try to [avoid passive voice in our docs](https://docs.microsoft.com/en-us/style-guide/grammar/verbs#active-and-passive-voice). I think it's fine to use active voice with `they` as the pronoun though.

```suggestion
    /// algorithms, they should also set an explicit key storage location.
```

Reply:
Good to know, and sorry for the false hint.

Reply:
No prob. Passive voice is used in my documentation content all the time. 😉 😛 

I have to remind myself of the style guidance frequently :)